common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -280,7 +280,7 @@ function (Sjson, AsyncProcess) {
         *     false: No caption file was specified, or an empty string was
         *         specified for the Youtube type player.
         */
-        fetchCaption: function () {
+        fetchCaption: function (fetch_with_youtubeId) {
```

Comment:
Can we update the doc strings for this method to show the newly added behaviour.